andela/travel_tool_front
# #161975169 Change badge on the request modal upon approval

#### What does this PR do?
Fixes issue badge on the request modal that does not change for approved requests

#### Description of Task to be completed?
- An approved request should have the ```Travel Stage``` badge.

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- checkout the branch `git checkout ch-change-badge-for-approved-requests-161975169`
- Start the server with `make start`
- Start the backend of this project using this link [Click here](https://github.com/andela/travel_tool_back)
- Visit https://travela-local.andela.com
- Login to the application
- Navigate to the request page
- Create a new request.
- The manager should approve the request or change the request to approved in the DB.
- View the request, the badge should show ```Travel Stage```

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161975169](https://www.pivotaltracker.com/story/show/161975169)

#### Screenshots (if appropriate)
<img width="1439" alt="screen shot 2018-11-15 at 18 01 39" src="https://user-images.githubusercontent.com/12511966/48561264-9db5a880-e900-11e8-846b-be9655a4cebf.png">


#### Questions:
